xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd

```diff
@@ -105,14 +105,14 @@ Rail transport, Roads and Road transport
 	<!-- ===ENTIITY IN VERSION IN FRAME====(Used in SITE FRAME)=================================================== -->
 	<xsd:complexType name="stopPlacesInFrame_RelStructure">
 		<xsd:annotation>
-			<xsd:documentation>Type for containment in frame of STOP PLACE, VEHICLE MEETING POINT, TAXI RANK.</xsd:documentation>
+			<xsd:documentation>Type for containment in frame of STOP PLACE</xsd:documentation>
 		</xsd:annotation>
 		<xsd:complexContent>
 			<xsd:extension base="containmentAggregationStructure">
 				<xsd:sequence>
-					<xsd:element ref="StopPlace_" maxOccurs="unbounded">
+					<xsd:element ref="StopPlace" maxOccurs="unbounded">
 						<xsd:annotation>
-							<xsd:documentation>Can be a STOP PLACE, VEHICLE MEETING POINT, TAXI RANK.</xsd:documentation>
+							<xsd:documentation>Can be a STOP PLACE.</xsd:documentation>
```

Comment:
```suggestion
							<xsd:documentation>A STOP PLACE.</xsd:documentation>
```

Reply:
@ue71603 The comment regarding Vehicile Meeting Point, is this something we are still missing.

Reply:
no.... all set in different elements.